lib/matplotlib/axis.py

```diff
@@ -1079,7 +1079,7 @@ def _get_tick_bboxes(self, ticks, renderer):
                 [tick.label2.get_window_extent(renderer)
                  for tick in ticks if tick.label2.get_visible()])
 
-    def get_tightbbox(self, renderer, *, ignore_label=None):
+    def get_tightbbox(self, renderer, *, for_layout_only=False):
```

Comment:
This should run through a deprecation and docs must be updated.

Reply:
The point is that this was only added in 3.3 and not released yet, so I'm trying to get this in before the 3.3 release.

Reply:
Still docs need to change the parameter name.

Reply:
indeed, done